.github/workflows/build.yml

```diff
@@ -28,3 +28,6 @@ jobs:
 
     - name: Build
       run: dotnet build VibeSomeMauiBro/VibeSomeMauiBro.csproj --configuration ${{ github.event_name == 'pull_request' && 'Debug' || 'Release' }}
```

Comment:
This should build the `vibe-some-maui-bro.sln` file, so the `--no-build` below will work.